packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -172,7 +172,10 @@ export const AddToCartModal = () => {
                                             <DisplayPrice
                                                 discountPriceProps={{as: 'p'}}
                                                 basePrice={lineItemBasePrice * quantity}
-                                                discountPrice={lineItemDiscountPrice * quantity}
+                                                discountPrice={
+                                                    typeof lineItemDiscountPrice === 'number' &&
+                                                    lineItemDiscountPrice * quantity
+                                                }
```

Comment:
Yeah, I think something like this (but you need to run it through lint, my code likely fails lint!)
```suggestion
                                                {...(typeof lineItemDiscountPrice === 'number') ?  { 
                                                    discountPrice: lineItemDiscountPrice * quantity
                                                } : {} }
```

Reply:
@bfeister please have another look. It is fixed. lint does not complain anymore. Thanks